modules/openideal_idea/openideal_idea.module

```diff
@@ -47,6 +48,18 @@ function openideal_idea_form_alter(&$form, FormStateInterface $form_state, $form
  * Implements hook_ENTITY_TYPE_presave().
  */
 function openideal_idea_node_presave(EntityInterface $entity) {
+  // Add status message for idea with draft or draft_approval state.
```

Comment:
Do we really need to add a message every time we save an idea?

I think we might have multiple saves before moving to "draft_approval" state, so maybe we should react on transition from "draft" to "draft_approval". What do you think?

Reply:
I think it's the right approach because both states (draft/draft_approval) need to be approved by the admin/moderator.